persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -58,12 +60,18 @@ public enum Command {
         ADD,
         GENERATE,
         DB_PUSH,
-        MIGRATE
+        MIGRATE,
+        PULL
     }
 
     private static final PrintStream errStream = System.err;
-    public static final String GENERATED_SOURCES_DIRECTORY = Paths.get("build", "generated-sources").toString();
-    public static final Path RESOURCES_EXPECTED_OUTPUT = Paths.get("src", "test", "resources", "test-src", "output")
+    public static final String GENERATED_SOURCES_DIRECTORY =
+            Paths.get("build", "generated-sources").toString();
```

Comment:
```suggestion
    public static final String GENERATED_SOURCES_DIRECTORY = Paths.get("build", "generated-sources").toString();
```

Reply:
This was fixed in https://github.com/ballerina-platform/persist-tools/pull/330/commits/fae659160549b702db4474c10a964b37c90dbd25